# Sensor data
std_msgs/Header header
string sensorDataInfo
uint8 size
float64[] sensorData
